src/mem3/src/mem3_rep.erl

```diff
@@ -118,6 +121,34 @@ make_local_id(SourceThing, TargetThing, Filter) ->
     <<"_local/shard-sync-", S/binary, "-", T/binary, F/binary>>.
 
 
+make_purge_id(SourceUUID, TargetUUID) ->
+    <<"_local/purge-mem3-", SourceUUID/binary, "-", TargetUUID/binary>>.
```

Comment:
The purge docid for mrview has version of purge implementation in it [see here](https://github.com/apache/couchdb/pull/1370/commits/c67815e8272ef2e90aa6a2a07522a9590bbbc49a#diff-2bd31f00dcfba780c6c65bc69c256854R46). Should we embed version here as well?

Reply:
I added versioning in 94fcf71 for mem3/replication case.

Reply:
Based on discussion with @davisp, removing all versioning for local purge document and change is reflected in 407b1dc for mem3 and mrview.

Reply:
That versioning thing was something that was planned for a rollback but after more development it became clear that we can't really do a parity rollback so there's no reason to try and code it. Due to how we want to transform the current purge data structure into a tree there's no obvious way to undo a tree into the existing structure without a whole bunch of code that would be exceedingly rarely executed. Given that complexity tradeoff it seems a lot more useful to just test the upgrade thoroughly.